scripts/testfiles

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
```

Comment:
This file is not currently executable. You should run `chmod +x scripts/testfiles.`